app/jobs/alma_full_dump_transfer_job.rb

```diff
@@ -0,0 +1,76 @@
+require 'net/sftp'
+
+class AlmaFullDumpTransferJob < ActiveJob::Base
+  queue_as :default
+
+  REMOTE_BASE_PATH = "/home/alma".freeze
+
+  def perform(dump:, job_id:, type_constant: 'BIB_RECORDS')
+    dump_file_type = DumpFileType.find_by(constant: type_constant)
+    Net::SFTP.start(sftp_host, sftp_username, password: sftp_password) do |sftp|
```

Comment:
suggestion: I wonder if extracting this to a little Downloader would make this easier to parse? Something like
```
Downloader.files_for(job_id: job_id).each do |file|
end
```

Not totally necessary, but might make this job easier to parse?